examples/request-animation-frame/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+use std::cell::RefCell;
+use std::rc::Rc;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen::JsCast;
+
+fn window() -> web_sys::Window {
+    web_sys::window().expect("no global `window` exists")
+}
+
+fn request_animation_frame(f: &Closure<FnMut()>) {
+    window()
+        .request_animation_frame(f.as_ref().unchecked_ref())
+        .expect("should register `requestAnimationFrame` OK");
+}
+
+fn document() -> web_sys::Document {
+    window()
+        .document()
+        .expect("should have a document on window")
+}
+
+fn body() -> web_sys::HtmlElement {
+    document().body().expect("document should have a body")
+}
+
+// This function is automatically invoked after the wasm module is instantiated.
+#[wasm_bindgen(start)]
+pub fn run() -> Result<(), JsValue> {
+    let f = Rc::new(RefCell::new(None));
+    let g = f.clone();
+
+    {
+        let mut i = 0;
+        *g.borrow_mut() = Some(Closure::wrap(Box::new(move || {
```

Comment:
Would be really helpful to have an explanatory comment or two on this line. No pressure, and no doubt the only non-core-Rust thing there is the `Closure::wrap()` ... but still, it would help to get what's going on (like why are we re-assigning it just after we assigned it to a clone?)

Reply:
Certainly! How's [this look](https://github.com/rustwasm/wasm-bindgen/commit/fcee4656923a1128d0a2faf781242ee2d7835db4)?

Reply:
Excellent, thanks!